src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -401,6 +409,18 @@ public DeferredBuildMessage(string text, MessageImportance importance)
             }
         }
 
+        // TOOD: Add commment
```

Comment:
TODO...

I'm also wondering if these DeferredResponseFiles should be more general...DeferredBuildMessage? Then we could reuse it in the future. If you think that's overengineering, go ahead and ignore me 🙂